Arm/Insts/Common.lean

```diff
@@ -54,15 +54,32 @@ def ConditionHolds (cond : BitVec 4) (s : ArmState) : Bool :=
   else
     result
 
+/-- Is the bit-vector x a-bit aligned? -/
+def Aligned (x : BitVec n) (a : Nat) : Prop :=
+  match a with
+    | 0 => True
+    | a' + 1 => (extractLsb a' 0 x) = (BitVec.zero _)
```

Comment:
```suggestion
def Aligned (x : BitVec n) (a : Nat) : Prop :=
  match a with
  | 0 => True
  | a' + 1 => extractLsb a' 0 x = BitVec.zero _
```

Indent `match` guards to be at the same indent level, and remove unnecessary parens